README.md

```diff
@@ -177,18 +177,22 @@ c.close()
 Create topics:
 
 ```python
-    new_topics = [NewTopic(topic, num_partitions=3, replication_factor=1) for topic in topics]
-    # Call create_topics to asynchronously create topics, a dict
-    # of <topic,future> is returned.
-    fs = a.create_topics(new_topics)
-
-    # Wait for each operation to finish.
-    for topic, f in fs.items():
-        try:
-            f.result()  # The result itself is None
-            print("Topic {} created".format(topic))
-        except Exception as e:
-            print("Failed to create topic {}: {}".format(topic, e))
+from confluent_kafka.admin import AdminClient, NewTopic
+
+a = AdminClient({'bootstrap.servers': 'mybroker'})
+
+new_topics = [NewTopic(topic, num_partitions=3, replication_factor=1) for topic in ["topic1", "topic2"]]
```

Comment:
We should use the default replication factor of 3 to not trick people into using replication_factor=1

Reply:
there's a tradeoff here between making sure there is zero friction for people running examples on a dev machine and best practice for running in production. I would argue the former is more important because by the time people get to the latter they've already adopted kafka and they're not going to stop due to this misunderstanding (but every point of friction early on is a risk of giving up). I'll add a comment instead, which I think is good enough to address your concern. The other way to do this would be push people towards a docker setup script for local testing that has 3 brokers (this seems optimal to me, but we don't have it yet).